test/integration/host_test.rb

```diff
@@ -117,6 +117,8 @@ def assert_interface_change(change, &block)
           assert modal.find('.interface_provision').checked?, "Provision checkbox is checked"
 
           modal.find(:button, "Cancel").click
+          # wait until modal is closed
+          page.has_css?('#interfaceModal', :visible => false)
```

Comment:
does this actually cause a wait/block somehow?  I'm not very familiar with capybara, but would assume it'd be similar to the table.find line below anyway.